### DISCUSSION TITLE
Is it possible to start a vm when the network adapter is not available?

### DISCUSSION THREAD

**Discussion by TheFern2** [utmapp/UTM#5034] [UTM Help (macOS)] [answered]
I have a mac m1, and I use a usb ethernet when I am at home. Sometimes I don't have the usb ethernet adapter with me when I am the office and the vm won't start because the adapter is missing and not connected. Only way is to remove the network completely, but then when I need it I have to re add it again. Is there a way to ignore the missing adapter?

**Comment by ktprograms** [comment, marked as answer]
I don't think there's any way to have "fallback" virtual network devices currently if that's what you're asking, but fyi you can just change the interface being used (if you're using Bridged)

You might also want to consider using Shared Networking if the VM doesn't need to access anything on the host network (you can still connect to/from the host machine itself)

**Reply by TheFern2** [reply]
Ah ok cool, yeah is bridged because it interacts with other devices on that subnet not just the host machine. I'll remove for now and use as an usb adapter inside the vm.
